neural_lifetimes/models/nets/event_model.py

```diff
@@ -9,14 +9,14 @@
 
 # TODO rename
 class EventEncoder(nn.Module):
-    def __init__(
-        self,
-        emb: CombinedEmbedder,
-        rnn_dim: int,
-        drop_rate: float = 0.0,
-    ):
+    def __init__(self, emb: CombinedEmbedder, rnn_dim: int, drop_rate: float = 0.0, num_layers: int = 1):
         super().__init__()
         self.emb = emb
+
+        if num_layers == 1:
+            drop_rate = 0.0
+            print("Dropout for RNN was set to 0, because num_layers=1.")
+
```

Comment:
Has this improved the performance?

Reply:
No. This a) allows to set a custom number of layers and b) takes care of a warning message during training that gets printed at every step.

The default num_layer is and was 1.